include/fmt/base.h

```diff
@@ -146,6 +146,8 @@
 // Use the provided definition.
 #elif defined(__GNUC__) && !defined(__EXCEPTIONS)
 #  define FMT_USE_EXCEPTIONS 0
+#elif defined(__clang__) && (!defined(__cpp_exceptions) || __cpp_exceptions < 199711L)
```

Comment:
I wonder why they check against 199711L?

Reply:
IDK. As I mentioned, this is how Clang does that C++ exceptions check.

Reply:
I suggest removing the `__cpp_exceptions < 199711L` part then if it works in your environment.

Reply:
Yes, it works.